portal-api/api-tests/v1/utilisation-report-service/previous-reports.api-test.js

```diff
@@ -31,23 +31,23 @@ describe('GET /v1/previous-reports/:bankId', () => {
         bank,
         month: 1,
         year,
-        dateUploaded: new Date(year, 0),
+        dateUploaded: new Date(year, 0).toISOString(),
```

Comment:
I'm slightly concerned about doing it this way as the reports we actually feed into the database use a `Date` object for `dateUploaded` rather than an ISO string. It might be better to use a function in line 88 to convert the dates in `expectedResponse` to their respective ISO strings. Alternatively, we can use a function in `JSON.parse` to re-convert ISO strings to `Date` objects like this https://stackoverflow.com/questions/4511705/how-to-parse-json-to-receive-a-date-object-in-javascript